com.nomachine.nxplayer.yml

```diff
@@ -0,0 +1,60 @@
+app-id: com.nomachine.nxplayer
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: launch_nxplayer
+tags:
+  - proprietary
+
+finish-args:
+  - --device=dri
+  - --filesystem=home
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+
+modules:
+  - name: nomachine
+    buildsystem: simple
+    build-commands:
+      - install -D NMLogo.png $FLATPAK_DEST/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png
```

Comment:
where is that file?

Reply:
https://www.nomachine.com/NMLogo.png

Reply:
so how does it makes its way to the build?

Reply:
nevermind it's there.